### PR TITLE
 Add debian package building

### PR DESCRIPTION
**Pull request by zarix908**
Adding configure files for debian package building with dpkg-buildpackage util. Build command: `./build.sh --deb`. Building requires entering sign key. I can create ppa repository on launchpad. Issue #90.

### CODE REVIEW

**Comment by snassar-podigee** [comment]
Is there a problem with the PR that is preventing getting this merged and debian packages built?

**Comment by zarix908** [comment]
@snassar-podigee  didn't know themselves, how a package should be organized

**Comment by usernamedt** [comment]
> Is there a problem with the PR that is preventing getting this merged and debian packages built?

> @snassar-podigee maintainers didn't know themselves, how a package should be organized

Currently, there is no single WAL-G binary, so initially, we need a PR that will introduce the uniform WAL-G rule-them-all binary so we can ship the deb package with it.

This was also discussed in https://github.com/wal-g/wal-g/issues/1006

**Comment by snassar-podigee** [comment]
It is completely OK if you choose not to invest your time in packaging wal-g, I was curious as to what the hold up is.

I will focus on compiling wal-g on debian myself.

**Comment by chipitsine** [comment]
is this superceeded by https://github.com/wal-g/wal-g/pull/1972 ?

**Comment by boosterKRD** [comment]
> is this superceeded by #1972 ?

 I don't think so. My proposal includes both Launchpad and packages for all Binaries, using GitHub Pages. I have listed the pros and cons in the pull request. I am ready to implement it while the community has not added anything there, but in principle I can force it and do it. My solution assumes the possibility of storage in GitHub Pages. And this is the history of package versions, not only for Postgres, but for everything else that we compile, roughly speaking. There is also a ready-made solution for Launchpad, so  I still advocate for my PR.  Ж) 
 
 Long Story Short : My Pull Request #1972  has been tested many times, and it works with both GitHub Pages and Launchpad. There will be no problems with implementation. All information about what it offers and how it works is provided in my Pull Request with screenshots.

**Comment by chipitsine** [comment]
if this PR brings its own value, @boosterKRD  can you please review it ?
